ref-schema-url/test/regex.spec.js

```diff
@@ -0,0 +1,52 @@
+const regex = new RegExp(
+  '(https?://[a-z0-9/\\.\\-]+[\\.]com)/([a-z0-9\\-]+)/(v\\d+\\.\\d+\\.\\d+(-\\d+)?)/([a-z\\./\\-]+.json)',
+);
```

Comment:
i know that might be overkill, but to be sure that this regex is always aligned with the one from the definition, i would require the definition and use the value from `pattern`.

Reply:
Good idea 👍🏼